.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+<!-- If this PR adds a CEP, do not replicate information in the document. Give a brief summary and link to related issues and PRs. -->
+
+# Summary of Changes
+<!--- In one or more sentences, describe the PR you are submitting. -->
+
+
+# Related CEPs and Issues
+<!--- Reference any related CEPs or issues and describe how this PR contributes to or addresses the problem. -->
+
+
+# Associated Developers
+<!--- Please mention any developers who should be alerted of this PR. -->
+
+
+# Design Notes
+<!--- Describe any design decisions or trade-offs you made. -->
```

Comment:
I'm not sure how I feel about this section. I feel like this information should be in the summary of changes. Another section that I have seen for templates is type of change (e.g., bug fix, new feature, breaking change, etc.). Is that something we want to have instead of Design Notes?

Reply:
> Another section that I have seen for templates is type of change (e.g., bug fix, new feature, breaking change, etc.). Is that something we want to have instead of Design Notes?

Ideally (and we never quite live up to that ideal) each PR will be responding to an issue and the issue will indicate what type it is...

Reply:
I can understand wanting to include the information in the summary section. My thinking here is: if we don't explicitly ask for it, then the summary might just be "I did this thing." Whereas, if we ask for them to highlight design decisions or trade-offs, the author at least has to scroll past it before they make an issue. 

Maybe a clearer structure here would be to make this a subsection of the Summary of Changes?